packages/p2p/src/components/advertiser-page/block-user/block-user-overlay/block-user-overlay.scss

```diff
@@ -15,14 +15,12 @@
         opacity: 0.9;
         top: 0;
         left: 0;
-        width: 94rem;
+        width: 100%;
         z-index: 9999;
 
         @include mobile {
             border: none;
             border-radius: unset;
-            width: 100%;
-            height: 100%;
```

Comment:
Parent `&__wrapper` already includes `width: 100%` and `height: 100%`, refactoring this since its basically duplicated code